docs/local.md

```diff
@@ -35,7 +35,7 @@
 
  - [install **helm3**](https://helm.sh/docs/intro/install/) to deploy k8gb and related test workloads
 
- - [install **k3d**](https://k3d.io/#installation) to run local [k3s](https://k3s.io/) clusters
+ - [install **k3d**](https://k3d.io/#installation) to run local [k3s](https://k3s.io/) clusters (minimum 4.0 version is required)
```

Comment:
Consider to put k3d version requirement to https://github.com/AbsaOSS/k8gb/blob/1974e67d438f7c8608b43dbc5acb1f8fbad23060/Makefile#L105 so it will be available in `make help`

Reply:
@ytsarev better now? ```deploy-full-local-setup Deploy full local multicluster setup (req. k3d >= 4.0)```